### DISCUSSION TITLE
using retrieve endpoint using postman

### DISCUSSION THREAD

**Discussion by tmartinscopo** [serenity-rs/serenity#2980] [Q&A]
Hello
I'm trying to use endpoint retrieve through postman. How do I pass the entityId as a parameter?

![image](https://github.com/user-attachments/assets/aa79f6dc-0c68-49e9-81ec-44b51dca07ae)

Thank for you help

**Comment by GnomedDev** [comment]
You have got the wrong serenity. This is a Discord library for bots written in Rust.
